src/encoded/genomic_data_service.py

```diff
@@ -41,7 +47,7 @@ def parse_params(self):
             self.sort_order = 'desc' if desc else 'asc'
 
         self.filter_params = {}
-        for filter_ in RNA_GET_FILTERS:
+        for filter_ in RNA_GET_FACETS:
```

Comment:
Do you mean 
`for facet in RNA_GET_FACETS:`

Reply:
good catch, I will change the variable name